hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -1198,4 +1202,38 @@ public static com.hederahashgraph.api.proto.java.FileID fromPbj(FileID someFileI
                 .setFileNum(someFileId.fileNum())
                 .build();
     }
+
+    public static TopicID toPbj(com.hederahashgraph.api.proto.java.TopicID topicId) {
+        return TopicID.newBuilder()
+                .realmNum(topicId.getRealmNum())
+                .shardNum(topicId.getShardNum())
+                .topicNum(topicId.getTopicNum())
+                .build();
+    }
+
+    public static SignatureMap toPbj(com.hederahashgraph.api.proto.java.SignatureMap sigMap) {
```

Comment:
⚠️ Even just in unit tests, it's not safe to assume all [`SignaturePair`](https://github.com/hashgraph/hedera-protobufs/blob/main/services/basic_types.proto#L747)'s are using Ed25519 signatures.

I'd suggest merging my `rm-protoc-and-compile-cons-svc` branch and making this method delegate to `protoToPbj(Objects.requireNonNull(sigMap), com.hederahashgraph.api.proto.java.SignatureMap.class)`.